gtk/gtk.go

```diff
@@ -9506,6 +9516,254 @@ func (v *TreeRowReference) Valid() bool {
 	return gobool(c)
 }
 
+/*
+ * GtkTreeSortable
+ */
+
+// TreeSortable is a representation of GTK's GtkTreeSortable
+type TreeSortable struct {
+	*glib.Object
+}
+
+// ITreeSortable is an interface type implemented by all structs
+// embedding a TreeSortable.  It is meant to be used as an argument type
+// for wrapper functions that wrap around a C GTK function taking a
+// GtkTreeSortable.
+type ITreeSortable interface {
+	toTreeSortable() *C.GtkTreeSortable
+}
```

Comment:
that one already exists as an interface

see `type TreeSortable interface` in line 9670 in master branch.  
you might want to merge the original one with yours

Reply:
I've just moved it, because I thought the place would match better :)

Reply:
Ah, I see